src/styles/stopwatch.scss

```
@@ -0,0 +1,69 @@
+.stopwatch {
+  margin: 100px;
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  width: $size;
+  height: $size;
+  border-radius: $size/2;
+  border: solid 1px black;
+}
+
+.stopwatch__center {
```

Використовуй вкладеність
```suggestion
&__center {
```